.azure/azure-pipelines.ci.yml

```diff
@@ -509,46 +510,121 @@ stages:
   - template: ./templates/run-wanperf.yml
     parameters:
       image: windows-2022
-      jobName: wanperf_5Mbps
```

Comment:
Please update `.\.github\workflows\wan-perf.yml` **instead** of this here. This is being deprecated.